cas/incident-report.md

```diff
@@ -1,169 +1,387 @@
 
-* 
+Effective ongoing commitments consist of the following:
 
-#### Where we got lucky
+1. **Specificity**: The commitment clearly outlines what the CA Owner will do, including the intended outcome, target completion timeframe, and any relevant metrics or indicators.
+ 
+2. **Actionability**: The commitment describes concrete steps the CA Owner will take, ensuring it's not merely a statement of intent or a vague promise.
+ 
+3. **Measurability**: The commitment includes a way to track progress and assess whether the CA has fulfilled its pledge. This could involve public reporting, audits, or other verifiable means.
+ 
+4. **Relevance**: The commitment directly relates to the incident report's findings and addresses the identified issues or concerns.
+ 
+5. **Impact**: The commitment aims to bring about positive change and improve the Web PKI ecosystem, either by preventing similar incidents or mitigating their potential impact.
 
-* 
+While both action items and ongoing commitments contribute to addressing the root cause(s), they are intended to serve distinct purposes and have unique characteristics:
 
-### Action Items
 
-| Action Item | Kind | Due Date |
-| ----------- | ---- | -------- |
-| Example | Prevent | 2038-01-19 |
+| Characteristic        | Action Item                    | Ongoing Commitment                             |
+| --------------------- | ------------------------------ | ---------------------------------------------- |
+| **Objective**         | Resolve the immediate incident | Prevent future incidents and improve practices |
+| **Scope**             | Narrow, specific tasks         | Broad, strategic initiatives                   |
+| **Timeframe**         | Short-term, with a deadline    | Long-term, continuous effort                   |
+| **Example**           | "We will revoke all mis-issued certificates within the next 48 hours. By that time, all revoked certificates will appear on the CRL located at $location." <br><br> "We will implement Version 1.0 of pkimetal in our production environment for pre-issuance linting by $date."  | "To reduce the likelihood of certificate mis-issuance, we will perform pre-issuance linting using pkimetal with the auto-detect profile set for all certificates issued after $date.  <br><br> Any error returned by pkimetal will result in blocking issuance and will necessitate manual review by at least two members of our compliance team.  <br><br> We will update pkimetal within 7 days after an updated release is made available.  <br><br> The presence of linting failures where certificates were issued more than 7 days after a new lint was released will represent a failure of this ongoing commitment." |
```

Comment:
Is this supposed to be an example of a good commitment? I see this commitment as outsourcing compliance of the CA to open-source linter projects, for a few reasons:

1. 7 days from linter release to production is far too rapid. This aggressive cadence does not afford time for a proper review of linter updates so that linter behavior is fully understood.
2. Linters do occasionally flag false positives. A CA that commits itself to stop issuance in the face of a linter error is painting itself into a corner.
3. A commitment to unconditionally action a linter's output is removing the agency from CA personnel to make their own decisions on interpretations of the BRs (and other documents). Linters are tools. They are not auditors.

Reply:
This table has been removed in #187.